src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="Header">
+      <a
+        href="#Home"
+        class="Image"
+      >
+        <img
+          class="ImageLogo"
+          src="images/logo.png"
+          alt="logo Moyo"
+        />
+      </a>
+      <nav class="Nav">
+        <ul class="ListLinks">
+          <li>
+            <a
+              href="#"
+              class="Link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="Link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="Link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li data-qa="hover">
+            <a
+              href="#"
+              class="Link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="Link"
```

Comment:
```suggestion
              class="link"
```
don't capitalize class names
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

Reply:
@lerastarynets , I made the changes

Reply:
@lerastarynets something is missing?